src/main/java/edu/gatech/cs2340/trydent/TrydentException.java

```diff
@@ -12,5 +12,8 @@
     public TrydentException(String message) {
         super(message);
     }
+    public TrydentException(String message, Exception nestedException) {
```

Comment:
nit: I like to have line-breaks between methods/constructors. But not a big deal, and others may or may not agree.